src/lib/validator.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Values to be validated
+ */
+export interface IValidationValue {
+  value: undefined | null | string;
+  error: string;
+}
+
+/**
+ * Returns true of val is undefined, null or empty string.
```

Comment:
I believe this doc is incorrect. Should be `Returns true if val is not undefined, not null, and not an empty string.`

Reply:
accept